### ISSUE TITLE
Strong+PCB 

### ISSUE THREAD

**Issue by he010103** [michuanhaohao/reid-strong-baseline#60]
@michuanhaohao 
I try to add PCB in the reid-strong-baseline. BNNeck is added in the feature of every part. But the performance is not better than the original reid-strong-baseline version. Is there any tricks when adding PCB? Thanks!

**Comment by michuanhaohao** [comment]
@he010103 Please read our new paper: [https://arxiv.org/pdf/1906.08332.pdf](https://arxiv.org/pdf/1906.08332.pdf). PCB achieves better performance on DukeMTMC-reID. We train PCB only with ID loss and BN layers.

![image](https://user-images.githubusercontent.com/23547900/61759484-2cb58180-adfb-11e9-9d74-47d80fcd8794.png)

**Comment by magicffourier** [comment]
@michuanhaohao I also try PCB and AlignedReID++.
In the paper, you said

> Our baseline boosts the performance of k-reciprocal reranking, PCB, AligedReID++, and CamStyle by a large margin

I found you learning rate is not similar with PCB and AlignedReID++. And I also get bad performance .
Could you please explain it.
